addons/web/static/src/js/common_env.js

```diff
@@ -29,6 +29,16 @@ odoo.define("web.commonEnv", function (require) {
     // Build the basic env
     const env = {
         _t,
+        browser: Object.assign({}, window, {
```

Comment:
I am not sure what we end up saying about this. I remember that it was not owl's job to include this is the generic env. What about Odoo though ?

Reply:
Last time we talked about this we decided to not do object assign like here, but to define only the useful keys manually, which we still need to change here.

Reply:
Will change with this: https://github.com/odoo-dev/odoo/commit/5943e38370b32292620711ee4184eb5acf4e2187